docs/src/accessibility-testing-js.md

```diff
@@ -44,7 +44,7 @@ test.describe('homepage', () => { // 2
 
     const accessibilityScanResults = await new AxeBuilder({ page }).analyze(); // 4
 
-    expect(accessibilityScanResults.violations).toEqual([]); // 5
+    expect(accessibilityScanResults.violations).toStrictEqual([]); // 5
```

Comment:
I think usually we recommend `toEqual` in these cases - could you change it to these?